app/views/words/show.html.erb

```diff
@@ -7,8 +7,8 @@
   <% @reviews.each do |review| %>
     <%= review.rating %> <%= review.body %>
     <% if current_user.try(:admin?) || current_user == review.user %>
-      <%= link_to "Edit Review", edit_word_review_path(review) %>
-      <%= link_to "Delete Review", word_review_path(review), method: :delete %>
+      <%= link_to "Edit Review", edit_word_review_path(@word, review) %>
```

Comment:
I assume you mean in the word controller? If that's the case, then @word is being set through a Controller Filter on line 2 of the controller since `@word =  Word.find(params[:id])` is being called on several actions, it made sense to have a before_action method for each of those actions.